worksheet_A/TerminalHacking.cpp

```diff
@@ -7,6 +7,34 @@
 const int wordLength = 5;
 const int numberOfWords = 15;
 
+// Returns number of matching letters
+int getLikeness(std::string guessedWord, std::string secretWord)
+{
+	int result = 0;
+
+	for (int i = 0; i < secretWord.length(); i += 1) {
+
+		if (secretWord.at(i) == guessedWord.at(i)) {
+			result += 1;
+		}
+	}
+
+	return result;
+}
+
+// Checks to see if word is in the list
+bool wordInList(std::string guess, std::set<std::string> options)
+{
+	for each (std::string word in options) {
```

Comment:
Apply a consistent formatting style. I.e. if the rest of the code has the `{` on the next line, so should yours